core/src/main/scala/com/microsoft/azure/synapse/ml/logging/SynapseMLLogging.scala

```diff
@@ -76,20 +76,34 @@ trait SynapseMLLogging extends Logging {
     logBase("constructor", None)
   }
 
-  def logFit[T](f: => T, columns: Int): T = {
-    logVerb("fit", f, columns)
+  def logFit[T](f: => T, columns: Int, logCertifiedEvent: Boolean = true,
```

Comment:
You don’t need an extra map for certified events args, if you use these APIs they will be auto populated

Reply:
Sorry, didn't quite get what you meant. How would the caller for logFit, logTransform, etc pass in the certified event attributes? Or did you mean remove the map parameter from logFit, but within it's body create one and pass to logVerb?

Reply:
they will be automatically populated using the scheme we discussed, it will automatically have the feature be SynapseML and the activity be fit, constructor, or transform

Reply:
I guess you meant things like dataframe row and column count will go to the attributes. Let's chat about this sometime today to get an accurate ask.